flink-filesystems/flink-azure-fs-hadoop/src/test/java/org/apache/flink/fs/azurefs/AzureFileSystemBehaviorITCase.java

```diff
@@ -85,9 +93,9 @@ private static boolean isHttpsTrafficOnly() throws IOException {
             return true;
         }
 
-        Assume.assumeTrue(
-                "Azure storage account not configured, skipping test...",
-                !StringUtils.isNullOrWhitespaceOnly(ACCOUNT));
+        assumeThat(ACCOUNT)
+                .describedAs("Azure storage account not configured, skipping test...")
+                .isNotBlank();
```

Comment:
I rewrote the logic here so that all of the `assumeThat` are more explicit.  I'm pretty sure it's equivalent, but I'd appreciate a bit of extra review on this change!